app/src/main/java/com/island/island/Activities/FeedActivity.java

```diff
@@ -46,6 +58,15 @@
     private RecyclerView.LayoutManager mLayoutManager;
     private List<Post> mArrayOfPosts;
     private SwipeRefreshLayout refreshLayout;
+    private CoordinatorLayout mainLayout;
```

Comment:
I noticed the lower case m prefix for member variables on some of our code and the stuff in the Udacity course. In this class we are using it only sometimes. I am not familiar with Android conventions to know if it's for all variables or only certain types of variables.